### ISSUE TITLE
WordPress FAQ Manager

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2038]
**Source**
Curtis

**Describe the bug**
WordPress FAQ Manager has been effectively abandoned and has many PHP errors. None of them are security issues, just updates in coding standards so older features are no longer supported and development setups with debugging spit a bunch of error messages everywhere.

The biggest issue for me is I have to "fix" it to be able to see other error messages that are actual bugs or issues in our code or else there are too many errors to effectively figure out what's going on.

**To Reproduce**
1. Turn on error logging
2. See lots of error output on your screen

**Expected behavior**
No errors.

**Options**
I do know the developer of the plugin so we could see about ProudCity taking it over and making the fixes. They'll only take 20 minutes but we would need to start looking at the WP forums on the plugin. We'd also need to have some plan to update it for the block editor at some point. I currently have a "fixed" version on my computer.

Another option is looking for a new FAQ plugin. We'd need to check the data storage and display on the new plugin to make sure that we don't lose customer data and so that the new one fits our theme. I expect this will take more time initially but less time long term since we wouldn't have to maintain the plugin.

@kevindherman @lukefretwell

**Comment by lukefretwell** [comment]
@curtismchale is this a dupe of #2031 ?

**Comment by curtismchale** [comment]
@lukefretwell yes it is. Clearly I fixed it once and then we never got around to discussing it by the time I needed to "fix" it again.

I closed the other issue since this has more context.
